src/api/nodeRpcCall.js

```diff
@@ -1362,29 +1362,68 @@ const createProposalAndVote = async (threshold, proposalContent, vote) => {
   return [proposal, voteAye];
 };
 
-const congressSendLlm = async ({ walletAddress, transferToAddress, transferAmount }) => {
+const congressProposeSpend = async ({ walletAddress, blockNumber, spendProposal }) => {
   const api = await getApi();
 
   const threshold = await congressMajorityThreshold();
-
-  const executeProposal = api.tx.llm.sendLlm(transferToAddress, transferAmount);
+  const remark = api.tx.llm.remark('Congress spending');
+  const transferAndRemark = api.tx.utility.batchAll([spendProposal, remark]);
+  const dayToBlocks = (24 * 60 * 60) / 6;
+  const motionDuration = 7 * dayToBlocks;
+
+  // CouncilAccount execute forces Congress to use scheduler with at least 4 days delay.
+  // We add 1 extra day in case of any desync between signing and sending tx.
+  const transferAtBlock = blockNumber + (5 * dayToBlocks) + motionDuration;
```

Comment:
Just a note, as we may want to make this smarter in some future PR.

So in this PR it's hardcoded:
1. 7 days for motion duration
2. 1 day for someone to execute the motion after voting ends
3. 4 days for Senate to potentially veto

That's for the worst-case scenario that we didn't reach the threshold of signatures before motion ended and the vote of prime member must be used to fill for non-voters.
However, if congress coordinates and gets the threshold sooner, we only need 4 days + the time needed for congress to coordinate, which could be as low as an hour or a day.
The UX for that is hard to get right though. Maybe we could allow proposer to specify the delay manually and only enforce 4d minimum?